qutip/core/cy/qobjevo.pyx

```diff
@@ -42,7 +42,7 @@ cdef class QobjEvo:
 
       * a :obj:`~Qobj` (which creates a constant :obj:`~QobjEvo` term).
 
-      * a list of such callables, pairs or :obj:`~Qobj`s.
+      * a list of such callables, pairs or :obj:`~Qobj`\s.
```

Comment:
What does the `\s` do?

Reply:
When not there, sphinx raise a `Inline literal start-string without end-string` warning, see:
https://stackoverflow.com/questions/58633196/restructuredtext-inline-code-followed-by-s-results-in-inline-interpreted-text

Reply:
:+1: